### PR TITLE
show federated cloud id in the personal settings

### PR DESCRIPTION
**Pull request by schiessle**
part of https://github.com/owncloud/core/issues/16190

cc @MorrisJobke

### CODE REVIEW

File: apps/files_sharing/templates/settings-personal.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+/** @var OC_L10N $l */
+/** @var array $_ */
+?>
+<div id="fileSharingSettings" class="section">
+	<h2><?php p($l->t('Federated Cloud'));?></h2>
+
+	<p>
+		Your Federated Cloud Id: <?php p($_['cloudId']); ?>
```
Comment (LukasReschke) [inline apps/files_sharing/templates/settings-personal.php]: L10n? :)

**Comment by karlitschek** [comment]
nice :+1:

**Comment by MorrisJobke** [comment]
I right trimmed `/` from the ID (`morris@cloud.yourhost.com` instead of `morris@cloud.yourhost.com/`) and added the l10n of the text.

:+1: from my side

**Comment by DeepDiver1975** [comment]
@owncloud-bot retest this please

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**